metron-deployment/amazon-ec2/tasks/check-hosts.yml

```diff
@@ -16,5 +16,5 @@
 #
 ---
 - name: Wait for connectivity to host(s)
-  local_action: wait_for host={{ inventory_hostname }} state=started timeout=300 delay=10
+  local_action: wait_for host={{ inventory_hostname }} state=started port=22 timeout=300 delay=10
```

Comment:
Yeah, that last line was exactly what was failing in rare occasions before. Still have no idea how. Wouldn't have believed it if I hadn't seen it myself.